superset/assets/src/visualizations/nvd3/utils.js

```diff
@@ -165,11 +165,17 @@ export function generateBubbleTooltipContent({
   return s;
 }
 
-export function hideTooltips(element) {
-  if (element) {
-    const targets = element.querySelectorAll('.nvtooltip');
-    if (targets.length > 0) {
-      targets.forEach(t => t.remove());
+export function hideTooltips(shouldRemove) {
```

Comment:
May be add a bit more comment what `shouldRemove` means

Reply:
👍